src/main/java/com/arcbees/bourseje/server/api/AdminResource.java

```diff
@@ -62,4 +64,12 @@ public Response setVoteState(VoteState voteState) {
 
         return Response.ok().build();
     }
+
+    @POST
+    @Path(ResourcesPath.CANDIDATE)
+    public Response addCandidate(Candidate candidate) {
```

Comment:
I think the REST resources could be rewritten a bit to return objects instead of Response. You could also use the ResourceDelegate. Not sure if it's worth it for this project at this point though